db/migrate/20170217085547_add_type_to_cloud_object_store_container.rb

```diff
@@ -0,0 +1,17 @@
+class AddTypeToCloudObjectStoreContainer < ActiveRecord::Migration[5.0]
+  class CloudObjectStoreContainer < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    add_column :cloud_object_store_containers, :type, :string
+    add_index :cloud_object_store_containers, [:type, :ems_id]
+
+    CloudObjectStoreContainer.reset_column_information
+    CloudObjectStoreContainer.update_all :type => 'CloudObjectStoreContainer'
```

Comment:
I don't understand the need for this data migration.  If CloudObjectStoreContainer is the base class, then you can leave the values as nil (This is what AR will do anyway if you create a new record from the base class).  You only need a data migration if you want to move the existing records to a known, existing subclass.